mastracode/src/main-v1.ts

```
@@ -0,0 +1,226 @@
+
+import { createMastraCodeAnalytics } from './analytics.js';
+import { isStreamDestroyedError } from './error-classification.js';
+import { createBrowserFromSettings, loadSettings } from './onboarding/settings.js';
+import { detectTerminalTheme } from './tui/detect-theme.js';
+import { MastraTUI } from './tui/index.js';
+import { applyThemeMode, restoreTerminalForeground } from './tui/theme.js';
+import { setupDebugLogging } from './utils/debug-log.js';
+import { drainPipedStdin, reopenStdinFromTTY } from './utils/stdin-pipe.js';
+import { releaseAllThreadLocks } from './utils/thread-lock.js';
+import { getCurrentVersion } from './utils/update-check.js';
+import { createMastraCode } from './index-v1.js';
```

ESLint

[error] `./index-v1.js` import should occur before import of `./onboarding/settings.js`

(import/order)